media/js/zamboni/files.js

```diff
@@ -403,8 +407,8 @@ function bind_viewer(nodes) {
                         }
 
                         if (metadata.sub_packages) {
-                            for (var prefix in metadata.sub_packages) {
-                                process_files(prefix, metadata.sub_packages[prefix]);
+                            for (var sb_prefix in metadata.sub_packages) {
+                                process_files(prefix, metadata.sub_packages[sb_prefix]);
```

Comment:
What's the 'sb'? If it's sub maybe 'subPrefix'? 

Imho we really should try and stop doing underscores for js (apart from JSON keys)